tests/pytests/test_fd_freq_energy.py

```diff
@@ -0,0 +1,84 @@
+import pytest
+import psi4
+import numpy as np
+
+from utils import compare_values
+from addons import using, uusing
+
+pytestmark = [pytest.mark.psi, pytest.mark.api]
```

Comment:
fyi, you could combine the tests into a test_molsym.py if you liked. the reason to not have generic top level variables like `molecules` `ref_freqs` etc. is that when the pytests get run 20 threads at a time, these can collide between files and cause obscure errors.